classes/settings_provider.php

```diff
@@ -148,18 +148,15 @@ public static function get_requiresafeexambrowser_options($context = null) : arr
         $options[self::USE_SEB_NO] = get_string('no');
         $options[self::USE_SEB_CONFIG_MANUALLY] = get_string('seb_use_manually', 'quizaccess_seb');
 
-        // @codingStandardsIgnoreStart
-        // TODO: Implement following features and uncomment options.
-        //if ($context && has_capability('quizaccess/seb:manage_seb_sebconfigtemplate', $context)) {
-        //    $options[self::USE_SEB_TEMPLATE] = get_string('seb_use_template', 'quizaccess_seb');
-        //}
+        if ($context && has_capability('quizaccess/seb:manage_seb_templateid', $context)) {
```

Comment:
Should we check if there is at least one template available?

Reply:
Or alternatively we can add extra validation to push the users to pick a template

Reply:
Or maybe both?